src/test/java/feiner/projectile/ProjectileTest.java

```diff
@@ -0,0 +1,59 @@
+package feiner.projectile;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class ProjectileTest {
+
+    @Test
+    public void testGetX() {
```

Comment:
`getX()`

You don't need to put the word "test" in front of all your test methods anymore. That is an old and outdated convention.